plugin/chatgpt.vim

```diff
@@ -170,6 +170,8 @@ function! SendHighlightedCodeToChatGPT(ask, context)
     endif
   elseif a:ask == 'review'
     let prompt = 'I have the following code snippet, can you provide a code review for?' . "\n" . yanked_text . "\n"
+  elseif a:ask == 'document'
+    let prompt = 'Identify language from this code snippet and returns documentation following language pattern' . "\n" . yanked_text . "\n"
```

Comment:
We can use the file format to explicitly pass the syntax to the prompt, it'll likely give us better results.
```suggestion
    let syntax = &syntax
    let prompt = 'Given the following code snippet written in ' . syntax . ' return documentation following language pattern conventions' . "\n" . yanked_text . "\n"
```